### PR TITLE
Implement drug metabolite within AMD

### PR DESCRIPTION
**Pull request by JohanWBorg**
Possibility of specifying "drug_metabolite" modeltype within the AMD tool which will run structsearch for drug_metabolite.
Also updated structsearch for drug_metabolite to always return a drug metabolite model by using a "plain" type model as base.

In addition to these implementations. Functionality for extracting CL, V and Q parameters connected to a metabolite has been introduced for accurate extraction of all parameters of interest when running drug metabolite models.

### CODE REVIEW

File: src/pharmpy/tools/structsearch/drugmetabolite.py [hunk 1]
```diff
@@ -4,6 +4,28 @@
 from pharmpy.modeling import add_metabolite, add_peripheral_compartment, set_name
 
 
+def create_base_drug_metaboltie(model: Model) -> Model:
```
Comment (rikardn) [inline src/pharmpy/tools/structsearch/drugmetabolite.py:7]: misspelled function name

File: src/pharmpy/tools/structsearch/drugmetabolite.py [hunk 2]
```diff
@@ -4,6 +4,28 @@
...
+        Input model for search
+
+    Returns
+    -------
+    Model
+        Base model for drug metabolite model search
+
+    """
+    model = add_metabolite(model)
+    model = set_name(model, "plain")
```
Comment (rikardn) [inline src/pharmpy/tools/structsearch/drugmetabolite.py:24]: Perhaps name "base" to match other tools or alternatively "base-metabolite"

File: src/pharmpy/tools/structsearch/tool.py [hunk 3]
```diff
@@ -83,6 +83,7 @@ def run_tmdd(context, model):
 
     wf = create_fit_workflow(qss_candidate_models)
     task_results = Task('results', bundle_results)
+    # FIXME : wf (Workflow) has no attribute called add_task
```
Comment (rikardn) [inline src/pharmpy/tools/structsearch/tool.py:86]: This is correct and needs to be fixed. I won't work to use `add_task` without going via a `WorkflowBuilder`

Comment (rikardn) [reply src/pharmpy/tools/structsearch/tool.py:86]: But since it is not related to this PR so no need to fix here.

**Review by rikardn — CHANGES_REQUESTED** [review]
Some comments

File: src/pharmpy/tools/structsearch/drugmetabolite.py (hunk 1 above)
Comment (JohanWBorg) [reply src/pharmpy/tools/structsearch/drugmetabolite.py:7]: Oh how embarrassing! I corrected the error.

File: src/pharmpy/tools/structsearch/drugmetabolite.py (hunk 2 above)
Comment (JohanWBorg) [reply src/pharmpy/tools/structsearch/drugmetabolite.py:24]: named "base_metabolite" for clarity of origin

File: src/pharmpy/tools/structsearch/tool.py (hunk 3 above)
Comment (JohanWBorg) [reply src/pharmpy/tools/structsearch/tool.py:86]: Will leave the comment as is for future fix then

**Comment by rikardn** [comment]
Thanks!

**Pull request merged**